grpc/src/main/java/com/linecorp/armeria/internal/client/grpc/ArmeriaClientCall.java

```diff
@@ -91,7 +92,8 @@
 final class ArmeriaClientCall<I, O> extends ClientCall<I, O>
         implements Subscriber<DeframedMessage>, TransportStatusListener {
 
-    private static final Runnable NO_OP = () -> {};
+    private static final Runnable NO_OP = () -> {
+    };
```

Comment:
revert? 😄 (Sorry about the formatter making this)

Reply:
🙇‍♂️